controller/users_blackbox_test.go

```diff
@@ -50,6 +50,51 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestObfuscateUserAsServiceAccountBadRequest() {
```

Comment:
@corinnekrych can you please use subtest to structure these tests? I know existing tests aren't structured in a sub-tests manner but it would be great if you could do it.
```go
func (s *TestUsersSuite) TestObfuscateUser(){
	s.T().Run("As a service account", func(t *testing.T){
		t.Run("ok", ...)
		t.Run("bad request", ...)
		t.Run("not found", ...)
	})
	s.T().Run("Not a service account", func(t *testing.T){
		t.Run("unauthorized", ...)
		t.Run("forbidden", ...)
	})
}
```

Reply:
@corinnekrych  @jarifibrahim I think either way this is fine. With or without subtests :)

Reply:
Yeah. Either way is fine :)

Reply:
But that's what subtests are for.